keyboards/gmmk/pro/rev1/ansi/keymaps/apazzy/keymap.c

```diff
@@ -0,0 +1,171 @@
+        KC_LCTL, KC_LGUI, KC_LALT,                            KC_SPC,                             KC_RALT, MO(WIN_FN),   KC_RCTL, KC_LEFT, KC_DOWN, KC_RGHT
+    ),
+
+    [WIN_FN] = LAYOUT(
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          KC_MJST,
+        _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,          _______,
+        _______, _______, _______, _______, _______, _______, _______, KC_PGUP, KC_UP,   KC_PGDN, KC_PSCR, _______, _______, KC_RST2,          _______,
+        _______, _______, _______, _______, _______, _______, KC_HOME, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______,          _______,          _______,
+        _______,          _______, _______, _______, _______, _______, KC_END,  NK_TOGG, _______, _______, _______,          _______, _______, KC_HOME,
+        _______,KC_WINLCK,_______,                            _______,                            _______, _______, _______, _______, _______, _______
+    ),
+
+};
+
+#ifdef ENCODER_MAP_ENABLE
+    const uint16_t PROGMEM encoder_map[][NUM_ENCODERS][2] = {
+        [0] = { ENCODER_CCW_CW(KC_VOLD, KC_VOLU) },
+        [1] = { ENCODER_CCW_CW(KC_TRNS, KC_TRNS) }
+    };
+#endif
+
+void matrix_scan_user(void) {
+    if (mouse_jiggle_mode) {
+        tap_code(KC_MS_UP);
+        tap_code(KC_MS_DOWN);
+        tap_code(KC_MS_LEFT);
+        tap_code(KC_MS_RIGHT);
+        tap_code(KC_MS_WH_UP);
+        tap_code(KC_MS_WH_DOWN);
+    } else {
+    }
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    switch (keycode) {
+        case KC_RST2:
+            if (record->event.pressed) {
+                reset_keyboard();
+            }
+            return false;
+		// GUI lock toggle
+        case KC_WINLCK:
+            if (record->event.pressed) {
+                keymap_config.no_gui = !keymap_config.no_gui; //toggle status
+            }
+            return false;
+        // Mouse jiggle toggle
+        case KC_MJST:
+            if (record->event.pressed) {
+                mouse_jiggle_mode = !mouse_jiggle_mode;
+            }
+            return false;
+        case KC_MJCX:
+            if (record->event.pressed) {
+                mouse_jiggle_mode = false;
+            }
+            return false;
+        case KC_ESC:
+            if (record->event.pressed) {
+                if (mouse_jiggle_mode) {
+                    mouse_jiggle_mode = false;
+                    return false;
+                }
+                return true;
+            }
+            return true;
+        case EMO_SHRUG:
+            if (record->event.pressed) {
+                SEND_STRING("¯\\_(ツ)_/¯");
+            }
+            return false;
+		// process all other keycodes normally
```

Comment:
This won't work — `SEND_STRING()` only supports ASCII characters.
```suggestion
```

Reply:
Would need unicode for this, specifically.